src/app/components/home/home.component.ts

```diff
@@ -8,16 +8,12 @@ import { FormControl, FormGroup } from '@angular/forms';
  * Import the ngrx configured store
  */
 import { Store } from '@ngrx/store';
-import { AppState } from './../store/appState.store';
+import { AppState } from '../../store/appState.store';
 
 @Component({
   selector: 'ae-home',
-  template: `
-    <div [formGroup]="messageForm">
-       <h1>{{messageForm.controls['messageText'].value}}</h1>
-       <input type="text" formControlName="messageText"/>
-    </div>
-    `
+  template: require('./home.component.html').toString(),
+  styles: [require('./home.component.scss').toString()],
```

Comment:
Maybe you can try this Maybe you can try this? https://github.com/AngularClass/angular2-webpack-starter/blob/master/src/app/home/home.component.ts#L17 Putting the Styles in an array in StylesUrl